config/config.proto

```diff
@@ -81,6 +81,13 @@ message FlowdockConfig {
   optional bool send_resolved = 4 [default = false];
 }
 
+message WebhookConfig {
```

Comment:
Docstring above this (and the FlowdockConfig above, seems it was forgotten there too).